numba/tests/test_refop_pruning.py

```diff
@@ -129,7 +130,11 @@ def func(n):
             self.check(func, (types.intp), basicblock=True, diamond=False,
                        fanout=True, fanout_raise=False)
 
-    @TestCase.run_test_in_subprocess
+    # FIXME: Checks in the test need to updated to account for new PM changes.
```

Comment:
Do you think this is fixable for this PR?

Reply:
I think so, I'll update the tests

Reply:
I have updated the test-check but since I'm not familiar with the internals of `refprune` pass this will need another set of eyes.